### ISSUE TITLE
Slack link has expired 

### ISSUE THREAD

**Issue by mahakg290399** [awslabs/python-deequ#228]
**Describe the bug**
The slack link to have a conversation with the developers, is not working. Maybe it is expired 

**To Reproduce**
1. Go to 'documentation'
2. Click on 'Jiin slack'
3. See error in your browser 

**Expected behavior**
After clicking on the link, it should forward you to join page on slack

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by chenliu0831** [comment]
Yeah there's no plan to bring it back. Use https://github.com/awslabs/python-deequ for now and we will remove the reference from readme.
